variant.go

```diff
@@ -119,6 +119,25 @@ func (v Variant) format() (string, bool) {
 		}
 		buf.WriteByte('}')
 		return buf.String(), unamb
+	case reflect.Struct:
+		if rv.NumField() == 0 {
+			return "()", false
+		}
+		unamb := true
+		var buf bytes.Buffer
+		buf.WriteByte('(')
+		for i := 0; i < rv.NumField(); i++ {
+			s, b := MakeVariant(rv.Field(i).Interface()).format()
+			unamb = unamb && b
+			fmt.Println(unamb)
+			buf.WriteString(s)
+			if i < rv.NumField()-1 {
```

Comment:
Actually the GVariant format still prints a comma for a single-value tuple, similar to Python. I know that I'm in parallel arguing that this format isn't supposed to be used for interchange, but I still think it's valuable to stick to what the other main implementation does.